addons/source-python/packages/source-python/listeners/tick.py

```diff
@@ -109,19 +109,17 @@ def add(self, delay):
 class Delay(WeakAutoUnload):
     """Execute a callback after a given delay."""
 
-    def __init__(
-        self, delay, callback, cancel_on_map_end=False, *args, **kwargs
-    ):
+    def __init__(self, delay, callback, cancel_on_map_end=False, args=(), kwargs={}):
```

Comment:
"Better practice" as in your favourite? ^^ I do realize that threading uses `kwargs=None`, and there's absolutely nothing wrong with that approach! I even said it's fine:

> I find it more clear to use a dictionary directly than `None`, not that there's anything wrong with `None` either.

Tbh you're just claiming that it's "better" practice and my practice is bad, yet you have no arguments other than "someone might get confused"... Someone could very easily mistake any implementation, mine works and is perfectly valid so it's just a matter of opinion. I do know that `None` is used more often, but it's highly due to not having to convert the arguments afterwards in the method with `self.x = dict(x)`.